TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:16.967, 248816322900, datacollector.dll, DataCollectorMain.Run: Starting data collector run with args: --port,55086,--parentprocessid,6796,--diag,C:\Users\dzene\Source\Repos\VVS_zadaca2.6\Hypo Banka\Unit Testovi\bin\Debug\fine-code-coverage\_outputfolder\diagnostics.datacollector.20-12-02_15-49-16_70365_4.log,--tracelevel,4
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:16.977, 248816363915, datacollector.dll, DataCollector: Monitoring parent process with id: '6796'
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:16.992, 248816510461, datacollector.dll, Trying to connect to server on socket : 127.0.0.1:55086 
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.017, 248816760182, datacollector.dll, SocketCommunicationManager : SetupClientAsync : Attempting to connect to the server.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:17.040, 248816996365, datacollector.dll, Connected to the server successfully 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:17.040, 248816998700, datacollector.dll, Using the buffer size of 16384 bytes
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.171, 248818304028, datacollector.dll, DataCollector: Start Request Processing.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.177, 248818366749, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Datacollector received message: (DataCollection.BeforeTestRunStart) -> {
  "SettingsXml": "<RunSettings><RunConfiguration><ResultsDirectory>C:\\Users\\dzene\\Source\\Repos\\VVS_zadaca2.6\\Hypo Banka\\Unit Testovi\\bin\\Debug\\fine-code-coverage\\_outputfolder</ResultsDirectory><TargetPlatform>X64</TargetPlatform><TargetFrameworkVersion>.NETCoreApp,Version=v3.1</TargetFrameworkVersion><DesignMode>False</DesignMode><CollectSourceInformation>False</CollectSourceInformation><TestAdaptersPaths>C:\\Program Files\\dotnet\\sdk\\3.1.201\\Extensions</TestAdaptersPaths></RunConfiguration><LoggerRunSettings><Loggers><Logger friendlyName=\"blame\" enabled=\"True\" /><Logger friendlyName=\"Console\" uri=\"logger://microsoft/TestPlatform/ConsoleLogger/v1\" assemblyQualifiedName=\"Microsoft.VisualStudio.TestPlatform.CommandLine.Internal.ConsoleLogger, vstest.console, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a\" codeBase=\"C:\\Program Files\\dotnet\\sdk\\3.1.201\\vstest.console.dll\" enabled=\"True\" /></Loggers></LoggerRunSettings><DataCollectionRunSettings><DataCollectors><DataCollector friendlyName=\"blame\" enabled=\"True\"><Configuration><ResultsDirectory>C:\\Users\\dzene\\Source\\Repos\\VVS_zadaca2.6\\Hypo Banka\\Unit Testovi\\bin\\Debug\\fine-code-coverage\\_outputfolder</ResultsDirectory></Configuration></DataCollector></DataCollectors></DataCollectionRunSettings></RunSettings>",
  "Sources": [
    "C:\\Users\\dzene\\Source\\Repos\\VVS_zadaca2.6\\Hypo Banka\\Unit Testovi\\bin\\Debug\\fine-code-coverage\\Unit Testovi.dll"
  ]
}
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.259, 248819180195, datacollector.dll, TestPluginCache: Update extensions started. Skip filter = False
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.260, 248819192439, datacollector.dll, TestPluginCache: Using directories for assembly resolution 'C:\Program Files\dotnet\sdk\3.1.201\Extensions'.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.260, 248819194046, datacollector.dll, TestPluginCache: Updated the available extensions to 'C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll,C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll'.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.271, 248819301078, datacollector.dll, TestPluginCache.DiscoverTestExtensions: finding test extensions in assemblies endswith: Collector.dll TPluginInfo: Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectorConfig TExtension: Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollector
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.271, 248819304726, datacollector.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.271, 248819306898, datacollector.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: 
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.271, 248819308235, datacollector.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.272, 248819318263, datacollector.dll, AssemblyResolver.ctor: Creating AssemblyResolver with searchDirectories C:\Program Files\dotnet\sdk\3.1.201\Extensions,C:\Program Files\dotnet\sdk\3.1.201
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.275, 248819341268, datacollector.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using extension path.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.275, 248819344972, datacollector.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.275, 248819347163, datacollector.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: 
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.275, 248819348380, datacollector.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.276, 248819350001, datacollector.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using allExtensionPaths: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.276, 248819352843, datacollector.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Program Files\dotnet\sdk\3.1.201\Extensions,C:\Program Files\dotnet\sdk\3.1.201
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.276, 248819354541, datacollector.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Program Files\dotnet\sdk\3.1.201\Extensions,C:\Program Files\dotnet\sdk\3.1.201
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.277, 248819362297, datacollector.dll, AssemblyResolver.OnResolve: Microsoft.TestPlatform.Extensions.BlameDataCollector: Resolving assembly.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.279, 248819384397, datacollector.dll, AssemblyResolver.OnResolve: Resolved assembly: Microsoft.TestPlatform.Extensions.BlameDataCollector, from path: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
TpTrace Error: 0 : 15388, 1, 2020/12/02, 15:49:17.280, 248819398094, datacollector.dll, The type "Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector" defined in "Microsoft.TestPlatform.Extensions.BlameDataCollector.dll" does not have ExtensionUri attibute.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.281, 248819408409, datacollector.dll, AssemblyResolver.OnResolve: Microsoft.TestPlatform.Extensions.EventLogCollector: Resolving assembly.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.283, 248819426567, datacollector.dll, AssemblyResolver.OnResolve: Resolved assembly: Microsoft.TestPlatform.Extensions.EventLogCollector, from path: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Error: 0 : 15388, 1, 2020/12/02, 15:49:17.287, 248819464970, datacollector.dll, The type "Microsoft.TestPlatform.Extensions.EventLogCollector.EventLogDataCollector" defined in "Microsoft.TestPlatform.Extensions.EventLogCollector.dll" does not have ExtensionUri attibute.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.288, 248819471322, datacollector.dll, TestPluginCache: Discovered the extensions using extension path 'C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll,C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll'.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.288, 248819472961, datacollector.dll, TestPluginCache: Discoverers are ''.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.288, 248819474132, datacollector.dll, TestPluginCache: Executors are ''.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.288, 248819475220, datacollector.dll, TestPluginCache: Setting providers are ''.
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:17.288, 248819476278, datacollector.dll, TestPluginCache: Loggers are ''.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.290, 248819497900, datacollector.dll, TestPluginManager.CreateTestExtension: Attempting to load test extension: Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector
TpTrace Error: 0 : 15388, 1, 2020/12/02, 15:49:17.291, 248819500666, datacollector.dll, The type "Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector" defined in "Microsoft.TestPlatform.Extensions.BlameDataCollector.dll" does not have ExtensionUri attibute.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.300, 248819595545, datacollector.dll, Listening on Endpoint : 127.0.0.1:55088
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.314, 248819731258, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : DataCollection started.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:17.390, 248820489248, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Datacollector received message: (DataCollection.TestHostLaunched) -> {
  "ProcessId": 15156
}
TpTrace Information: 0 : 15388, 6, 2020/12/02, 15:49:17.750, 248824097812, datacollector.dll, Using the buffer size of 16384 bytes
TpTrace Information: 0 : 15388, 6, 2020/12/02, 15:49:17.751, 248824102807, datacollector.dll, Accepted Client request and set the flag
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.535, 248831942601, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.584, 248832434862, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.585, 248832440850, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.662, 248833209901, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.667, 248833262412, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.689, 248833479197, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.693, 248833524722, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.709, 248833682265, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.709, 248833688384, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.710, 248833690978, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.712, 248833709191, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.712, 248833715086, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.712, 248833718748, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.713, 248833721246, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.714, 248833734728, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.715, 248833739957, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.715, 248833742019, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.737, 248833967232, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.742, 248834009489, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.742, 248834013376, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.742, 248834017175, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.745, 248834046521, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.746, 248834052701, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.746, 248834055470, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.752, 248834114008, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.753, 248834119749, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.753, 248834122684, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.753, 248834125065, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.755, 248834148743, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.756, 248834154493, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.756, 248834157689, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.759, 248834184719, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.760, 248834190158, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.760, 248834193137, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.760, 248834195525, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.762, 248834209898, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.762, 248834215166, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.762, 248834217284, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.764, 248834237489, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.765, 248834243198, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.765, 248834246349, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.765, 248834248724, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.767, 248834261854, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.767, 248834266791, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.768, 248834268860, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.774, 248834336314, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.775, 248834342515, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.775, 248834345825, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.775, 248834348670, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.777, 248834362727, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.777, 248834367904, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.778, 248834370142, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.781, 248834400704, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.781, 248834406668, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.782, 248834409617, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.782, 248834412037, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.783, 248834425638, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.784, 248834430712, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.784, 248834432794, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.786, 248834453579, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.787, 248834459076, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.787, 248834461930, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.787, 248834464795, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.848, 248835069079, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.848, 248835078129, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.849, 248835081879, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.852, 248835109943, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.852, 248835118671, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.853, 248835123359, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.853, 248835127195, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.855, 248835142244, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.855, 248835148401, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.856, 248835150901, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.860, 248835194872, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.861, 248835205081, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.862, 248835216906, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.863, 248835220098, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.864, 248835237744, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.865, 248835246569, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.866, 248835249837, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.868, 248835274342, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.869, 248835283461, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.869, 248835287997, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.870, 248835291666, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.879, 248835387190, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.880, 248835393155, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.880, 248835395627, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.884, 248835429966, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.884, 248835437655, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.885, 248835440903, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.885, 248835443864, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.886, 248835458476, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.887, 248835465171, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.887, 248835468674, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.890, 248835492339, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.890, 248835498455, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.891, 248835501533, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.891, 248835504321, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.892, 248835517989, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.893, 248835523806, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.893, 248835526274, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.897, 248835561226, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.897, 248835567390, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.898, 248835570574, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.898, 248835573556, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.900, 248835589343, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.900, 248835597956, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.901, 248835601147, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.904, 248835628881, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.904, 248835635120, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.904, 248835638299, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.905, 248835641919, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.907, 248835661266, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.908, 248835669946, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.908, 248835673222, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.910, 248835697417, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.911, 248835703604, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.911, 248835707878, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.912, 248835711025, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.913, 248835726992, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.914, 248835735429, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.914, 248835738149, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.917, 248835759027, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.917, 248835768663, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.918, 248835773541, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.918, 248835777478, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.923, 248835827783, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.924, 248835836946, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.925, 248835840308, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.936, 248835949838, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.937, 248835959378, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.937, 248835964386, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.938, 248835968966, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.941, 248836005155, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.942, 248836011101, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.942, 248836013610, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.954, 248836131203, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.955, 248836140649, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.955, 248836145644, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.956, 248836149576, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.972, 248836317001, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.973, 248836326319, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.974, 248836330035, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.977, 248836363404, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.978, 248836373733, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.978, 248836378692, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.979, 248836382794, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.981, 248836403135, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.982, 248836412948, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.982, 248836416863, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.985, 248836448041, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.986, 248836457594, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:18.987, 248836462573, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.987, 248836466971, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.989, 248836487930, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.990, 248836496907, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:18.991, 248836500310, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.000, 248836596695, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.001, 248836607141, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.002, 248836612455, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.002, 248836616581, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.004, 248836637707, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.005, 248836647718, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.006, 248836651448, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.010, 248836689728, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.011, 248836699165, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.011, 248836704471, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.011, 248836708784, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.014, 248836729722, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.015, 248836738935, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.015, 248836743035, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.022, 248836809099, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.023, 248836818867, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.023, 248836823533, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.023, 248836827621, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.034, 248836935839, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.035, 248836945128, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.036, 248836948965, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.042, 248837012814, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.043, 248837022155, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.043, 248837026894, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.044, 248837030788, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.049, 248837081693, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.050, 248837090938, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.050, 248837094757, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.054, 248837133399, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.055, 248837143357, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.056, 248837148910, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.056, 248837153040, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.065, 248837242718, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.066, 248837252141, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.066, 248837255551, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.069, 248837285617, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.070, 248837291441, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.070, 248837296713, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.071, 248837300517, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.077, 248837362243, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.082, 248837415321, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.083, 248837427553, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.087, 248837467038, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.091, 248837500252, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.092, 248837516997, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.093, 248837528384, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.096, 248837549612, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.097, 248837564723, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.098, 248837570352, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.100, 248837591915, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.101, 248837605238, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.102, 248837613439, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.103, 248837619823, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.104, 248837638822, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.106, 248837657549, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.107, 248837663498, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.109, 248837682210, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.110, 248837693953, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.111, 248837700581, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.111, 248837706275, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.113, 248837722553, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.114, 248837734358, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.115, 248837739159, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.117, 248837764500, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.118, 248837775739, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.119, 248837783211, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.120, 248837789208, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.122, 248837808978, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.123, 248837819426, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.123, 248837823149, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.125, 248837845162, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.126, 248837857997, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.127, 248837866650, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.128, 248837872899, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.129, 248837888697, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.131, 248837903097, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.131, 248837906651, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.133, 248837928766, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.135, 248837939080, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.135, 248837946032, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.136, 248837950289, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.137, 248837967215, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.139, 248837981443, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.139, 248837986152, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.141, 248838007829, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.143, 248838024206, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.144, 248838031982, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.144, 248838038441, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.147, 248838060057, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.148, 248838074708, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.149, 248838079952, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.154, 248838132042, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.155, 248838144512, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.156, 248838150563, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.156, 248838154807, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.165, 248838243389, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.166, 248838257589, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.167, 248838263073, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.173, 248838325150, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.174, 248838337233, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.175, 248838345993, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.176, 248838352098, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.181, 248838404448, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.182, 248838416012, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.183, 248838421473, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.188, 248838469933, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.189, 248838481719, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 15388, 4, 2020/12/02, 15:49:19.190, 248838489360, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.190, 248838494646, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 15388, 4, 2020/12/02, 15:49:19.218, 248838775157, datacollector.dll, DataCollectionTestCaseEventHandler: Test session ended
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.368, 248840272256, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Datacollector received message: (DataCollection.AfterTestRunEnd) -> false
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.369, 248840282429, datacollector.dll, Blame Collector : Session End
TpTrace Error: 0 : 15388, 1, 2020/12/02, 15:49:19.369, 248840286729, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.370, 248840289113, datacollector.dll, DataCollectionManager.CleanupPlugins: CleanupPlugins called
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:19.370, 248840290531, datacollector.dll, DataCollectionManager.CleanupPlugins: Cleaning up 1 plugins
TpTrace Verbose: 0 : 15388, 1, 2020/12/02, 15:49:19.370, 248840292806, datacollector.dll, dataCollectorInfo.DisposeDataCollector: calling Dispose() on Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.370, 248840294527, datacollector.dll, DataCollectionManager.CleanupPlugins: CleanupPlugins finished
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.370, 248840297835, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Session End message received from server. Closing the connection.
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.371, 248840300267, datacollector.dll, Closing the connection !
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.371, 248840301571, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : DataCollection completed
TpTrace Information: 0 : 15388, 1, 2020/12/02, 15:49:19.371, 248840303557, datacollector.dll, Program.Main: exiting datacollector process.
